runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/ExternalTest.java

```diff
@@ -139,8 +139,8 @@ public void expandPythonTest() {
           testPipeline
               .apply(Create.of("1", "2", "2", "3", "3", "3"))
               .apply(
-                  External.<KV<String, Integer>>of(
-                      "beam:transforms:xlang:count", new byte[] {}, target))
+                  External.of("beam:transforms:xlang:count", new byte[] {}, target)
```

Comment:
This still feels a bit odd, but it's good enough that I don't want to block the testing work. Could you please start a discussion on the dev list about what the best API should be (for {single,multiple}{input,output}?)